index.js

```diff
@@ -15,16 +15,16 @@ module.exports = fp(function (fastify, options, next) {
 
     // static salt to be used for key derivation, not great for security,
     // but better than nothing
-    var salt = 'mq9hDxBVDbspDR6nLfFT1g=='
+    var salt = 'mq9hDxBVDbspDR6n'
 
     if (options.salt) {
       salt = options.salt
     }
 
-    salt = Buffer.from(salt, 'base64')
+    salt = Buffer.from(salt, 'ascii')
```

Comment:
We should likely not do this is if `salt` is a buffer.

Reply:
We can check salt, If it is a buffer there is no need to `Buffer.from()`

Reply:
Exactly.